### PR TITLE
[Excalidraw Electron article] Add aside and use more inclusive language

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add aside
- Use more inclusive language

### CODE REVIEW

File: src/site/content/en/blog/deprecating-excalidraw-electron/index.md [hunk 1]
```diff
@@ -37,7 +41,7 @@ Soon after [@vjeux](https://twitter.com/vjeux) created the initial version of Ex
 January 2020 and [blogged about it](reflections-on-excalidraw/), he proposed the following in
 [Issue #561](https://github.com/excalidraw/excalidraw/issues/561#issue-555138343):
 
-> Would be great to wrap Excalidraw within Electron (or equivalent) and publish it as a native
+> Would be great to wrap Excalidraw within Electron (or equivalent) and publish it as a [platform-specific]
```
Comment (tomayac) [inline src/site/content/en/blog/deprecating-excalidraw-electron/index.md:44]: Making visible that I have altered the quote without changing its sense. This looks like a great solution.

File: src/site/content/en/blog/deprecating-excalidraw-electron/index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ description: |
 authors:
   - thomassteiner
 date: 2021-01-07
-updated: 2021-01-22
+updated: 2021-01-26
```
Comment (tomayac) [inline src/site/content/en/blog/deprecating-excalidraw-electron/index.md:15]:
```suggestion
updated: 2021-01-27
```

File: src/site/content/en/blog/deprecating-excalidraw-electron/index.md (hunk 1 above)
Comment (kaycebasques) [reply src/site/content/en/blog/deprecating-excalidraw-electron/index.md:44]: I like that approach

**Review by kaycebasques — APPROVED** [review]
Thank you for making these changes 🙏  I appreciate it a lot

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/deprecating-excalidraw-electron/index.md [hunk 3]
```diff
@@ -68,7 +72,7 @@ cross-platform desktop apps with JavaScript, HTML, and CSS"_. Apps built with El
 _"compatible with Mac, Windows, and Linux"_, that is, _"Electron apps build and run on three
 platforms"_. According to the homepage, the hard parts that Electron makes easy are
 [automatic updates](https://www.electronjs.org/docs/api/auto-updater),
-[native menus and notifications](https://www.electronjs.org/docs/api/menu),
+[system-level menus and notifications](https://www.electronjs.org/docs/api/menu),
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/deprecating-excalidraw-electron/index.md:75]: The proposal came from @petele, and I think in this case it actually works well. Especially on macOS it conveys the idea that the menu is on the top, where regular apps have their menu, too (and not in the app window).

[the remark this answers is not in the document]
Comment (petele) [reply src/site/content/en/blog/deprecating-excalidraw-electron/index.md:75]: essentially, provided by the lower level system, OS, app framework, etc. I think it works reasonably well here.

**Pull request merged**